src/lib/help.ts

```diff
@@ -7,9 +7,9 @@ import config from 'config';
 
 import * as commands from './commands';
 
-function formatCommands(cmds: { name: string; description: string; hidden: boolean; position: number }[]) {
+function formatCommands(cmds: { name: string; description: string; hidden: boolean }[]) {
   return cmds
-    .sort((a, b) => (a.position > b.position ? 1 : -1))
+    .sort((a, b) => (a.name > b.name ? 1 : -1))
```

Comment:
a lot of our behind-the-scenes logic here is such a mess lol. the field is called `command` in the command file but `name` here. got it.